test/metabase/api/session_test.clj

```diff
@@ -33,6 +34,15 @@
                         (reset! (:attempts throttler) nil))
                       (thunk)))
 
+(defn- mock-request []
+  (edn/read-string (slurp "test/metabase/api/sample-request.edn")))
```

Comment:
probably could use a delay here?

Reply:
I thought about that but I thought reading the file was fast enough that it wasn't worth the added memory usage in CI to cache it once it's read, since it's only used twice. I guess it doesn't make a big difference either way. I'll change it